packages-content-model/roosterjs-content-model-editor/lib/editor/createEditorCore.ts

```diff
@@ -64,9 +67,12 @@ export function createEditorCore(
         ...createStandaloneEditorDefaultSettings(options),
 
         environment: {
-            // It is ok to use global window here since the environment should always be the same for all windows in one session
             isMac: window.navigator.appVersion.indexOf('Mac') != -1,
```

Comment:
I think window can be undefined in Server side rendering like in NextJS. We had a similar report some time ago in the Browser utility in roosterjs editor dom, so we added a undefined check there. Probably not a big deal for this PR. but just an FYI

Reply:
Good point.

Browser object was initialized when the code is loaded, so it is possible to be null if do server side render.
Now in Content Model editor I'm removing the usage of Browser object, and use this environment object instead. It is initalized only when we create an editor, it is triggered by js code and should not be rendered from server side directly.